style.css

```diff
@@ -470,3 +482,376 @@ h2 {
+    color: #fff;
+    width: 63px;
+    height: 20px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .mail {
+    display: block;
+    width: 24px;
+    height: 24px;
+    background-image: url(./Images/Icon\ -\ Mail.png);
+    margin-left: 20px;
+    margin-top: 20px;
+    gap: 30px;
+  }
+
+  header {
+    width: 100%;
+    height: 8px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
+
+  .headline {
+    width: 100%;
+    height: 550px;
+    background-image: url(/Images/Illustration-header@2x.png);
+    background-size: 500px 768px;
+    background-position: 100% 30%;
+  }
+
+  .brands {
+    display: flex;
+    flex-direction: column;
+    list-style-type: none;
+    width: 20px;
+    height: 98.96px;
+    margin-top: 150px;
+  }
+
+  .icon {
+    display: block;
+    padding: 10px;
+  }
+
+  .headline-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 192px 0;
+    position: absolute;
+    width: 566px;
+    height: 500px;
+    left: 143px;
+    top: 72px;
+  }
+
+  .title {
+    display: block;
+    padding: 0;
+    margin-left: 220px;
+    width: 556px;
+    font-family: 'Inter', sans-serif;
+    font-size: 32px;
+    font-weight: 500;
+    line-height: 44px;
+    letter-spacing: 0;
+    text-align: left;
+    margin-top: auto;
+  }
+
+  p {
+    font-family: 'Inter', sans-serif;
+    font-size: 20px;
+    font-weight: 400;
+    line-height: 32px;
+    letter-spacing: 0;
+    text-align: left;
+    margin-top: 0;
+  }
+
+  .About-myself {
+    width: 100%;
+    height: 600px;
+  }
+
+  .Recent-work {
+    height: 52px;
+    width: 100%;
+    margin-top: 0;
+  }
+
+  .middle {
+    display: flex;
+    flex-direction: row;
+    margin-left: 150px;
+    gap: 24px;
+  }
+
+  .about-img {
+    background-image: url(/Images/img-Placeholder.png);
+    height: 448px;
+    width: 684px;
+    left: 0;
+    top: 0;
+  }
+
+  .rightblock {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 14px;
+    height: 384px;
+    margin-top: 50px;
+  }
+
+  .Multi-Post-ul {
+    height: 40px;
+    width: 297px;
+    left: 0;
+    top: 236px;
+    border-radius: 0;
+  }
+
+  .work-container {
+    height: 448px;
+    width: 100%;
+    border-radius: 0;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(2, 1fr);
+    gap: 20px;
+    justify-items: center;
+    align-items: center;
+  }
+
+  .art-container {
+    height: 435px;
+    width: 369px;
+    left: 0;
+    top: 0;
+    border-radius: 0;
+  }
+
+  .art-container1 {
+    height: 435px;
+    width: 369px;
+    background-image: url(/Images/Img\ Placeholder5.png);
+  }
+
+  .art-container2 {
+    height: 435px;
+    width: 369px;
+    background-image: url(/Images/Img\ Placeholder6.png);
+  }
+
+  .about {
+    width: 100%;
+    margin-top: 330px;
+    margin-left: 0;
+  }
```

Comment:
- [ ] Dear @arnoldnekemiah, as mentioned by the previous reviewer your `about` section needs some touch. Especially the button is not as shown in the figma design and the alignments are not set correctly. Please address this.
- [ ] Also, I notice after your last review, your section decoration image is missing. This is caused by your newer changes. Please try to add the decoration image. 
### Your about section 
![image](https://user-images.githubusercontent.com/69077061/222876991-01be57fd-d9dc-4b70-8d22-02d5a51ba235.png)
###Figma design 
 

![222708514-a5092ce6-44e6-47a8-90fc-2b2cc0a2d2d0](https://user-images.githubusercontent.com/69077061/222876998-f96bf05d-4338-47a1-a9d6-0ded64cce3ee.png)